src/Domain/CoordinateMaps/Wedge3D.cpp

```diff
@@ -109,14 +111,32 @@ std::array<tt::remove_cvref_wrap_t<T>, 3> Wedge3D::inverse(
   const ReturnType& physical_y = physical_coords[1];
   const ReturnType& physical_z = physical_coords[2];
 
+  if (physical_z < 0.0 or equal_within_roundoff(physical_z, 0.0)) {
+    return boost::none;
+  }
+
   const ReturnType cap_xi = physical_x / physical_z;
   const ReturnType cap_eta = physical_y / physical_z;
   const ReturnType one_over_rho =
       1.0 / sqrt(1.0 + square(cap_xi) + square(cap_eta));
 
-  // Using auto keeps this as a blaze expression.
-  const auto zeta_coefficient =
+  const ReturnType zeta_coefficient =
       (scaled_frustum_rate_ + sphere_rate_ * one_over_rho);
+  // If -sphere_rate_/scaled_frustum_rate_ > 1, then
```

Comment:
Might be easier to parse if written as `If -sphere_rate_ > scaled_frustum_rate_`

Reply:
That doesn't work:  `-a/b > 1`  is equivalent to `-a>b` only if `b` is positive; I think the comment as written works for all cases.